samples/layouts/avatar/icon/src/index.ts

```diff
@@ -1,6 +1,16 @@
-import 'igniteui-webcomponents';
-import './IconAvatarStyles.css'
+import 'igniteui-webcomponents/src/components/avatar/avatar';
+import 'igniteui-webcomponents/src/components/icon/icon';
+import { registerIcon } from 'igniteui-webcomponents/src/components/icon/icon.registry';
+import 'igniteui-webcomponents/src/styles/themes/material.css';
 
-export class IconAvatar {}
+export class IconAvatar {
+    constructor() {}
+}
+
+registerIcon(
```

Comment:
Move this code inside the constructor, so it gets transfered to the "browser" when running all the samples.
```suggestion
export class IconAvatar {
    constructor() {
       registerIcon("home",
           "https://unpkg.com/material-design-icons@3.0.1/action/svg/production/ic_home_24px.svg",
           "material"
       );
    }
}
```